package.json

```diff
@@ -60,31 +71,31 @@
   },
   "size-limit": [
     {
-      "path": "index.js",
+      "path": "source/index.js",
```

Comment:
You need to use build file here to check what is the size after Babel.

Reply:
doh, good point, these should be pointing at

`dist/cjs/index.js`

or

`dist/index.js`

not in `source`